modules/virt-adding-pci-device-hyperconverged-operator.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-adding-pci-device-hyperconverged-operator_{context}"]
+= Adding PCI devices to the HyperConverged Operator
+To enable the availability of PCI devices in the cluster, add details about the PCI devices to the HyperConverged Operator in the YAML file. The PCI devices are exposed and permitted to be used in the cluster.
```

Comment:
The first sentence is a little wordy; I would suggest something like:
s/HyperConverged Operator in the YAML file/HyperConverged object

Reply:
Updated the content.